client/src/app/+admin/users/user-list/user-list.component.ts

```diff
@@ -92,12 +103,47 @@ export class UserListComponent extends RestTable implements OnInit {
         }
       ]
     ]
+
+    this.columns = [
+      { key: 'username', label: 'Username' },
+      { key: 'email', label: 'Email' },
+      { key: 'quota', label: 'Video quota' },
+      { key: 'role', label: 'Role' },
+      { key: 'createdAt', label: 'Created' }
+    ]
+    this.selectedColumns = [...this.columns]
+    this.columns.push({ key: 'quotaDaily', label: 'Daily quota' })
+    this.columns.push({ key: 'pluginAuth', label: 'Auth plugin' })
+    this.columns.push({ key: 'lastLoginDate', label: 'Last login' })
   }
 
   getIdentifier () {
     return 'UserListComponent'
   }
 
+  getRoleClass (role: UserRole) {
+    switch (role) {
+      case UserRole.ADMINISTRATOR:
+        return 'badge-purple'
+      case UserRole.MODERATOR:
+        return 'badge-blue'
+      default:
+        return 'badge-yellow'
+    }
+  }
+
+  getColumn (key: string) {
+    return this.selectedColumns.find((col: any) => col.key === key)
+  }
+
+  getUserVideoQuotaPercentage (user: User & { rawVideoQuota: number, rawVideoQuotaUsed: number}) {
```

Comment:
Could we create a type for the results of `getUsers`? So you don't have to use `& { rawVideoQuota: number, rawVideoQuotaUsed: number}` each time